portfolio.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+<body>
+  <header>
+    <a href="" class="nameLink">John Doe</a>
+    <nav>
+      <img src="images/Icon-Menu.png" alt="Menu Icon" class='menuIcon'>
+    </nav>
+  </header>
+  <h1>Hey There. I'm Alex</h1>
+  <h2>I’am a Software Developer</h2>
+  <h3>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</h3>
```

Comment:
- [ ] How about using a (p tag element) and changing the fonts with style instead? 🤔